packages/react-components/react-alert/src/stories/AlertAction.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+
+import { ArrowUndo16Filled } from '@fluentui/react-icons';
+
+import { Alert } from '../index';
+
+export const Action = () => (
+  <>
+    <Alert
+      intent="success"
+      content="Message sent"
+      action={{
+        appearance: 'transparent',
+        children: 'Undo',
+      }}
+    />
+    <br />
```

Comment:
_nit:_ Might be better to use other ways of spacing things out like `display: flex` on the container plus `margins` on the children.